src/frida/repl.py

```diff
@@ -78,15 +89,37 @@ def _unload_script(self):
 
         def _process_input(self, reactor):
             self._print_startup_message()
-            self._ready.wait()
+            while self._ready.wait(0.5) != True:
+                if not reactor._running:
```

Comment:
Should not access this variable without holding the lock; sounds like the reactor needs an accessor for that?